src/crystal-fann/cascade_network.cr

```diff
@@ -2,13 +2,20 @@ module Fann
   module Network
     class Cascade
       property :nn
+      getter :input_size
+      getter :output_size
 
       def initialize(input : Int32, output : Int32)
         @output_size = output
         @input_size = input
         @nn = LibFANN.create_shortcut(2, input, output)
       end
 
+      def initialize(@nn)
```

Comment:
How about, for load we just have 2 initializes

1) regular
2) for string

```
new(5,5)
# or
new("load/from/this/path")
```
What do you think ?

Reply:
Nice suggestion! :+1: